codestyle/intellij-java-google-style.xml

```diff
@@ -0,0 +1,596 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<code_scheme name="GoogleStyle">
```

Comment:
i would rename it to `ButterStyle` to make it our own as we will probably modify it over time.